scripts/e2e-tests.sh

```diff
@@ -323,8 +323,8 @@ function setup_test_cluster() {
     abort "kubeconfig context set to ${k8s_cluster}, which is forbidden"
 
   # If cluster admin role isn't set, this is a brand new cluster
-  # Setup the admin role and also KO_DOCKER_REPO
-  if [[ -z "$(kubectl get clusterrolebinding cluster-admin-binding 2> /dev/null)" ]]; then
+  # Setup the admin role and also KO_DOCKER_REPO if it is a GKE cluster
+  if [[ -z "$(kubectl get clusterrolebinding cluster-admin-binding 2> /dev/null)" && "${k8s_cluster}" =~ ^gke_.* ]]; then
```

Comment:
I think we should also move definition of `k8s_user` down here too, it just prints irrelevant info if not gke

Reply:
I prefer making it explicit that this is gcloud user, as this PR doesn't remove the dependency on gcloud. In line 339, use "gcloud user is" instead of "User is".

Reply:
Works for me